awx/main/models/inventory.py

```diff
@@ -2261,27 +2234,11 @@ def build_script_private_data(self, inventory_update, private_data_dir):
 class openstack(PluginFileInjector):
     ini_env_reference = 'OS_CLIENT_CONFIG_FILE'
     plugin_name = 'openstack'
-    # minimum version of 2.7.8 may be theoretically possible
-    initial_version = '2.8'  # Driven by consistency with other sources
-
-    @property
-    def script_name(self):
-        return 'openstack_inventory.py'  # exception
 
-    def _get_clouds_dict(self, inventory_update, cred, private_data_dir, mk_cache=True):
+    def _get_clouds_dict(self, inventory_update, cred, private_data_dir):
         openstack_data = _openstack_data(cred)
 
         openstack_data['clouds']['devstack']['private'] = inventory_update.source_vars_dict.get('private', True)
-        if mk_cache:
```

Comment:
was `mk_cache` just a script thing? or is it a deprecated feature?

Reply:
I think @chrismeyersfsu might retain some memory of this ancient contract.

By default, it would save a serialization of the inventory to a file somewhere and persist it if less than 300 seconds had passed. But you couldn't just turn it off, so instead, we pointed it to a file we knew would be deleted at the end of the run, so that it would never be re-used in place of actually making API calls to the service.

Reply:
I remember there being bugs related to this